### ISSUE TITLE
error in replaceFirst

### ISSUE THREAD

**Issue by runefs** [jcoplien/trygve#84]
The below code doesn't replace the first occurrence of the given string. The third line seems related to the bug in split

```
{
   System.out.println("[".replaceFirst("[","g"));
   System.out.println("=".replaceFirst("=",""));
   System.out.println("g".replaceFirst("g","["));
   System.out.println("done");
}

/* GOLD
   g

   [
   done
*/
```

**Comment by jcoplien** [comment]
Fixed in 1.5.15.

**Comment by jcoplien** [comment]
Thanks again, Rune.
